src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
```

Comment:
Change name to be more consistent to get/setter standards
```suggestion
    public boolean getDone() {
```

Reply:
Changed! Thank you